Error running tests from graphql crates

## What went wrong? 😲
```
   Compiling async-graphql-actix-web v3.0.38
error[E0599]: no variant or associated item named `Http2Payload` found for enum `PayloadError` in the current scope
   --> /Users/jamesbrunskill/.cargo/registry/src/index.crates.io-6f17d22bba15001f/async-graphql-actix-web-3.0.38/src/request.rs:113:43
    |
113 | ...                   PayloadError::Http2Payload(e) if e.is_io() => e.into_io().unwrap(),
    |                                     ^^^^^^^^^^^^ variant or associated item not found in `PayloadError`

error[E0599]: no variant or associated item named `Http2Payload` found for enum `PayloadError` in the current scope
   --> /Users/jamesbrunskill/.cargo/registry/src/index.crates.io-6f17d22bba15001f/async-graphql-actix-web-3.0.38/src/request.rs:114:43
    |
114 | ...                   PayloadError::Http2Payload(e) => io::Error::new(ErrorKind::Other, e),
    |                                     ^^^^^^^^^^^^ variant or associated item not found in `PayloadError`

For more information about this error, try `rustc --explain E0599`.
```

## Expected behaviour 🤔

Tests should run!

## How to Reproduce 🔨

Steps to reproduce the behaviour:

Run `cargo test --package graphql_cold_chain --lib -- test_notifications --nocapture `

## Your environment 🌱
- rustc 1.76.0

--- a/server/Cargo.toml
+++ b/server/Cargo.toml
@@ -4,7 +4,7 @@ strip = true
 [workspace.dependencies]
 # dependencies used in graphql crates:
 actix-web = { version = "4.0.1", default-features = false, features = [
-  "macros",
+  "macros","rustls"
 ] }
 anymap = "0.12"
 async-graphql = { version = "3.0.35", features = ["dataloader", "chrono"] }

--- a/server/cli/Cargo.toml
+++ b/server/cli/Cargo.toml
@@ -20,7 +20,7 @@ server = { path = "../server" }
 graphql = { path = "../graphql" }
 
 anyhow.workspace = true
-async-graphql = "3.0.35"
+async-graphql = { workspace = true }
 clap = { version = "3.1.8", features = ["derive"] }
 chrono = { version = "0.4.23" }
 diesel = { version = "1.4.7", default-features = false, features = ["chrono"] }

--- a/server/graphql/plugin/Cargo.toml
+++ b/server/graphql/plugin/Cargo.toml
@@ -14,13 +14,11 @@ graphql_core = { path = "../core" }
 graphql_general = { path = "../general" }
 graphql_types = { path = "../types" }
 
-actix-web = { version = "4.0.1", default-features = false, features = [
-    "macros",
-] }
+actix-web = { workspace = true }
 anymap = "0.12"
 async-std = "1.9.0"
 async-graphql = { version = "3.0.35", features = ["dataloader", "chrono"] }
-async-graphql-actix-web = "3.0.35"
+async-graphql-actix-web = { workspace = true }
 async-trait = "0.1.30"
 chrono = { version = "0.4", features = ["serde"] }
 reqwest = { workspace = true }

--- a/server/graphql/programs/Cargo.toml
+++ b/server/graphql/programs/Cargo.toml
@@ -16,11 +16,10 @@ graphql_general = { path = "../general" }
 graphql_types = { path = "../types" }
 
 async-std = { workspace = true }
-
-actix-web = { version = "4.0.1", default-features = false, features = ["macros"] }
+actix-web = { workspace = true }
 anymap = "0.12"
-async-graphql = { version = "3.0.35", features = ["dataloader", "chrono"] }
-async-graphql-actix-web = "3.0.35"
+async-graphql = { workspace = true }
+async-graphql-actix-web = { workspace = true }
 async-trait = "0.1.30"
 chrono = { version = "0.4", features = ["serde"] }
 reqwest = { workspace = true } 

--- a/server/graphql/sensor/Cargo.toml
+++ b/server/graphql/sensor/Cargo.toml
@@ -15,7 +15,7 @@ util = { path = "../../util" }
 graphql_core = { path = "../core" }
 graphql_types = { path = "../types" }
 
-actix-web = { workspace = true }
+actix-web = { version = "4.0.1", features = ["rustls"] }
 anymap= { workspace = true }
 async-graphql = { workspace = true }
 async-graphql-actix-web = { workspace = true }

--- a/server/graphql/sensor/Cargo.toml
+++ b/server/graphql/sensor/Cargo.toml
@@ -15,8 +15,8 @@ util = { path = "../../util" }
 graphql_core = { path = "../core" }
 graphql_types = { path = "../types" }
 
-actix-web = { version = "4.0.1", features = ["rustls"] }
-anymap= { workspace = true }
+actix-web = { workspace = true }
+anymap = { workspace = true }
 async-graphql = { workspace = true }
 async-graphql-actix-web = { workspace = true }
 async-trait = { workspace = true }

--- a/server/service/Cargo.toml
+++ b/server/service/Cargo.toml
@@ -49,7 +49,7 @@ x509-parser = { version = "0.15", features = ["verify"] }
 actix-rt = "2.6.0"
 assert-json-diff = "2.0.1"
 httpmock = "0.6.6"
-actix-web = { version = "4.0.1" }
+actix-web = { workspace = true }
 tokio = { version = "1.21.1", features = ["macros", "rt-multi-thread", "time"] }
 
 [features]

